doc/source/serve/doc_code/vllm_example.py

```diff
@@ -0,0 +1,88 @@
+import json
+import random
+from typing import AsyncGenerator
+from starlette.requests import Request
+from starlette.responses import StreamingResponse, Response
+
+from transformers import AutoTokenizer
+
+from ray import serve
```

Comment:
[Nit] Can you move the `serve` import after the `vllm` imports. I want to highlight that we're using Ray Serve, and it sort of gets buried if we put it before the `vllm` imports.

Reply:
in general order should be:

builtin imports

followed by 3rd-party lib imports

followed by ray imports